content/github/collaborating-with-issues-and-pull-requests/changing-the-base-branch-of-a-pull-request.md

```diff
@@ -20,6 +20,13 @@ versions:
 4. In the base branch drop-down menu, select the base branch you'd like to [compare changes against](/github/committing-changes-to-your-project/comparing-commits#comparing-branches). ![Base branch drop-down menu ](/assets/images/help/pull_requests/pull-request-edit-base-branch.png)
 5. Read the information about changing the base branch and click **Change base**. ![Base branch change confirmation button ](/assets/images/help/pull_requests/pull-request-base-branch-confirm.png)
 
+{% tip %}
+
+**Tip:**
+When you open a Pull Request, GitHub will set the base to the commit that branch references. If the branch is updated in the future, GitHub will not update the base branch's commit.
```

Comment:
```suggestion
**Tip:** When you open a pull request, {% data variables.product.product_name %} will set the base to the commit that branch references. If the branch is updated in the future, {% data variables.product.product_name %} will not update the base branch's commit.
```
@Ruth-ikegah When we add a tip, note, or warning, we start the information on the same line as the callout, like in this [example](https://github.com/github/docs/blob/main/contributing/content-markup-reference.md#usage).

In the issue, pull request was uppercase, but it shouldn't be, so I swapped it out here.

Instead of writing out `GitHub`, we use a [product variable](https://github.com/github/docs/blob/main/contributing/content-style-guide.md#product-names). Even though the name GitHub should never change 😆 we use a variable the same way we do for all other products. You can see them all in the [product.yml](https://github.com/github/docs/blob/main/data/variables/product.yml).

Reply:
Looking at this one staging, can you bump the whole tip to the top of article? Maybe under the **Warning** box? It seems like users might want this information before they go through the whole procedural, so I think it'd be helpful to put it up top. Thank you ✨